src/sys/unix/waker.rs

```diff
@@ -51,7 +52,7 @@ mod fdbased {
 }
 
 #[cfg(all(
-    not(mio_unsupported_force_poll_poll),
+    not(any(target_os = "solaris", mio_unsupported_force_poll_poll)),
```

Comment:
Not needed.

Reply:
I got following error without this change:
```
error[E0432]: unresolved imports `self::fdbased`, `self::waker::Waker`
  --> src/sys/unix/mod.rs:24:20
   |
24 |     pub(crate) use self::waker::Waker;
   |                    ^^^^^^^^^^^^^^^^^^
   |
  ::: src/sys/unix/waker.rs:66:15
   |
66 | pub use self::fdbased::Waker;
   |               ^^^^^^^ could not find `fdbased` in `self`

```